src/test/server/Server_test.cpp

```diff
@@ -367,10 +370,98 @@ class Server_test : public beast::unit_test::suite
     }
 
     void
+    testBadConfig ()
```

Comment:
This test is outputting (expected) error messages. You could do something like [this](https://github.com/ripple/rippled/blame/b7e2a3bd5f0d5156893b1797fd2fb51ef5a33e0e/src/test/core/TerminateHandler_test.cpp#L36) to redirect cout in order to make the test less verbose.